operator-framework-core/src/main/java/io/javaoperatorsdk/operator/LeaderElectionManager.java

```diff
@@ -102,11 +102,15 @@ private void startLeading() {
   }
 
   private void stopLeading() {
-    log.info("Stopped leading for identity: {}. Exiting.", identity);
-    // When leader stops leading the process ends immediately to prevent multiple reconciliations
-    // running parallel.
-    // Note that some reconciliations might run for a very long time.
-    System.exit(1);
+    if (configurationService.getLeaderElectionConfiguration().orElseThrow().isExitOnStopLeading()) {
+      log.info("Stopped leading for identity: {}. Exiting.", identity);
+      // When leader stops leading the process ends immediately to prevent multiple reconciliations
+      // running parallel.
+      // Note that some reconciliations might run for a very long time.
+      System.exit(1);
+    } else {
+      log.info("Stopped leading, not exiting");
```

Comment:
```suggestion
      log.info("Stopped leading, configured not to exit");
```

Reply:
To make it more explicit that the operator doesn't exit on purpose.

Reply:
makes sense, thx!